knimin/tests/test_pm_plate_list.py

```diff
@@ -0,0 +1,56 @@
+from unittest import main
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class TestPMPlateListHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get(
+            '/pm_plate_list/')
+        self.assertEqual(response.code, 200)
+        self.assertTrue(
+            response.effective_url.endswith('?next=%2Fpm_plate_list%2F'))
+
+    def test_get(self):
+        self.mock_login_admin()
+        response = self.get('/pm_plate_list/')
+        self.assertEqual(response.code, 200)
+        # Test the page title
+        self.assertIn('Sample Plate List', response.body)
+        # Test the first sample plate
+        p1 = db.get_sample_plate_list()[0]
+        x = p1['fill'][1]
+        battery = ''
+        if x == 1.0:
+            battery = 'full'
+        elif x >= 0.667:
+            battery = 'quarters'
+        elif x > 0.333:
+            battery = 'half'
+        elif x > 0.0:
+            battery = 'quarter'
+        else:
+            battery = 'empty'
```

Comment:
This logic is a little confusing.  Is it possible to split up this unittest into 5 separate tests - each test with hard-coded battery values.